fleece/boto3.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import absolute_import
+
+import boto3 as real_boto3
+from botocore.client import Config
+
+DEFAULT_TIMEOUT = None
+DEFAULT_CONNECT_TIMEOUT = None
+DEFAULT_READ_TIMEOUT = None
+
+
+def set_default_timeout(timeout=None, connect_timeout=None, read_timeout=None):
+    """
+    The purpose of this function is to install default socket timeouts that
+    different than those used by boto3 and botocore. Clients obtained from the
+    `fleece.boto3.client()` function will have these timeouts set, unless
+    explicitly overriden.
+
+    Note that socket timeouts are automatically retried by boto, according to
+    its own retry policies.
+
+    :param timeout timeout for socket connections and reads. This is a
```

Comment:
I see that the units are documented in the readme, but not here. Could you add that? I had to look around a bit to realize that the units are seconds (rather than milliseconds, etc.).